services/client/src/components/footer/styles.js

```diff
@@ -2,25 +2,32 @@ import css from "styled-jsx/css";
 
 export default css`
   .footer {
+    position: fixed;
+    margin-left: 0;
+    width: 46rem;
```

Comment:
Applying a width results in the header being smaller or larger than expected when viewing at different screen widths: 
<img width="735" alt="screen shot 2018-09-14 at 20 27 15" src="https://user-images.githubusercontent.com/1443700/45571017-c962d680-b85c-11e8-9fcb-0dfd7cd6429d.png">